content/terms/premium-support.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: markdown
+title: Terms of Service for Sourcegraph Premium Support
+---
+
+Last Modified: May 5, 2023
+
+This document outlines the basic terms and conditions that apply to Sourcegraph's Premium Support Offerings.
+
+## Premium Support
+
+### Benefits of Premium Support
+Purchasing Premium Support provides the following services beyond those provided by Sourcegraph's standard support model:
+- **Accelerated Service Level Agreements (SLAs)** - Accelerated triaging and queue priority lead reduced times to resolution.
+- **Elevated Support Expertise** - Two named, dedicated senior support engineers possessing deep knowledge and understanding of you specific organizations setup and tech stack with the ability to assist in the resolution of issues that arise.
+- **Heightened Support via Slack** - Faster response times and closer collaboration through direct access to Sourcegraph experts with closer connection to end users and administrators.
+- **More Proactive Collaboration** - Hands-on technical guidance to ensure successful migrations, upgrades, and maintenance; early access to new features; quarterly insights reports.
+
+### Premium Support SLAs
+
+1st response times for both Standard and Premium Support remain the same according to our [SLAs](https://handbook.sourcegraph.com/departments/technical-success/support/#slas). However, Emergency and Severe Impact issues are supported 24 x 7 with Premium Support (versus 24 x 5 with Standard Support).
+
+### Premium Support Packages
+
+Each Premium Support Package is priced independently. For pricing, please discuss with your Sourcegraph contacts. The current Packages offered are:
+- Package 1: Premium SLAs
+- Package 2: Dedicated Support (note - purchasing <em>Package 2: Dedicated Support</em> comes with <em>Package 3: Slack Support</em>)
+- Package 3: Slack Support
```

Comment:
Remind me, does this mean _only_ support via slack? What about Sales/TA wanting to have a slack channel for comms? Where did we land there?

Reply:
We leaned towards TA as well but never got confirmation whether not having Premium Support package 2 meant a TA would not be there at all. cc @billCaplan